mathics/eval/tracing.py

```diff
@@ -26,8 +26,18 @@ def skip_trivial_evaluation(expr, status: str, orig_expr=None) -> bool:
     * the evaluation is a literal that evaluates to the same thing,
     * evaluating a Symbol which the Symbol.
     * Showing the return value of a ListExpression literal
+    * Evaluating Pattern[] which define a pattern.
```

Comment:
This change is inspired by looking at TraceEvaluation. There is no benefit in watching an evaluation of `_`,  `n_`,  or  `n_Integer`.